internal/cli/server_upgrade.go

```diff
@@ -297,8 +298,9 @@ func (c *ServerUpgradeCommand) Flags() *flag.Sets {
 		f.StringVar(&flag.StringVar{
 			Name:    "snapshot-name",
 			Target:  &c.snapshotName,
-			Default: "",
-			Usage:   "Filename to write the snapshot to.",
+			Default: defaultSnapshotName,
+			Usage: "Filename to write the snapshot to. If no name is specified, by" +
+				"default a timestamp will be appended to the default snapshot name.",
```

Comment:
Can you throw this change on `uninstall` too while you're merging  🙏🏻

Reply:
Ah yes, good call! :+1: